File: api/Models/LayAwaySchedule.cs
using System;

namespace ByzmoApi.Models
{
    public class LayAwaySchedule
    {
        public int Id { get; set; }
        public int OrderId { get; set; }
        public int ProductId { get; set; }
        public decimal Monthly { get; set; }
        public DateTime Date { get; set; }
        public bool IsNonRefundDeposit { get; set; }
        public bool IsCleared { get; set; }
        public bool IsShipping { get; set; }
        public bool IsSendEmail { get; set; }
        public bool IsPrSend { get; set; }
        public bool IsInsurance { get; set; }
        public PaymentMethod PaymentMethod { get; set; }
        public int PaymongoStatus  { get; set; }
    }
}

File: web/src/app/classes/layaway-schedule.ts
import { Moment } from 'moment';
import { FileViewer } from './file-viewer';
import { PaymentMethod } from './payment-method';

export class LayAwaySchedule {
    id: number;
    orderId: number;
    productId: number;
    date: Date;
    monthly: number;
    isNonRefundDeposit: boolean;
    isCleared: boolean;
    isShipping: boolean;
    attachment: FileViewer;
    isSendEmail: boolean;
    isPrSend: boolean;
    paymentMethod: PaymentMethod;
    isinsurance: boolean;
    paymongoStatus: number;
    constructor() {
        this.attachment = new FileViewer();
        this.paymentMethod = new PaymentMethod();
    }
}



